### PR TITLE
Home Page - meet the team scrolling avatars

### PR DESCRIPTION
**Pull request by TheFatPanda97**
### Ticket
[Ticket Link](https://trello.com/c/KPufkszY/17-implement-meet-our-team-section-from-designs)

### What is changed / added
- added scrolling avatars
- added vscode css custom directives

### Screenshots
PR

Notes:
1. Sorry for the relatively large PR, message me for any clarification 🙏
2. The choppiness in the gif is not present in real life, it is simply my attempt to reduce the gif size by lowering the framerate so it could be included in this PR

### CODE REVIEW

**Comment by TheFatPanda97** [comment]
[Implement "meet our team" section from designs](https://trello.com/c/KPufkszY/17-implement-meet-our-team-section-from-designs)

File: components/ScrollingAvatars/ScrollingAvatars.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  },
+  mid: {
+    size: 100,
+    margin: 30,
+  },
+  large: {
+    size: 150,
+    margin: 50,
+  },
+};
```
Comment (AaronCGoidel) [inline components/ScrollingAvatars/ScrollingAvatars.tsx:32]: I don't like calling this an `enum` since this is a POJO enums are a real TS feature

File: components/ScrollingAvatars/ScrollingAvatars.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+) => {
+  const currAvatarWidth = avatarInfos.length * (size + 2 * margin);
+  const desiredWidth = window.innerWidth + 20 * currAvatarWidth;
+  const remainingWidth = desiredWidth - currAvatarWidth;
+  const remainingAvatars = Math.ceil(remainingWidth / currAvatarWidth);
+  const appendedAvatarsInfos = [...avatarInfos];
+
+  for (let i = 0; i < remainingAvatars; i += 1) {
+    appendedAvatarsInfos.push(avatarInfos[i % avatarInfos.length]);
+  }
```
Comment (AaronCGoidel) [inline components/ScrollingAvatars/ScrollingAvatars.tsx:50]: if it's not a bother, this could be rolled into a `forEach`

just to stay consistent with ES6 everywhere possible

File: components/ScrollingAvatars/ScrollingAvatarsContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import React, { FC, useState } from 'react';
+import _ from 'underscore';
+
+import ScrollingAvatars from './ScrollingAvatars';
+
+interface AvatarInfo {
+  name: string;
+  position: string;
+  imgSrc: string;
+  key?: number;
```
Comment (AaronCGoidel) [inline components/ScrollingAvatars/ScrollingAvatarsContainer.tsx:10]: not sure we even need key here. I think using a unique string identifier is good and we have that in `name`. Plus, it saves on some logic below if we just key by `name`

it isn't exactly the same pattern but I would read [this](https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067)

File: components/ScrollingAvatars/ScrollingAvatarsContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      <ScrollingAvatars
+        key={key}
+        scrollDirection={key % 2 === 0 && !startScrollRight ? 'left' : 'right'}
+        avatarInfos={avatarInfos}
+        setName={setName}
+        setPosition={setPosition}
+      />
+    ));
+  } else {
+    let currKey = 0;
```
Comment (AaronCGoidel) [inline components/ScrollingAvatars/ScrollingAvatarsContainer.tsx:36]: see above comment on keys

File: components/ScrollingAvatars/ScrollingAvatarsContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    mappedScrollingAvatars = _.chunk(avatarInfos, avatarInfos.length / rows).map(
```
Comment (AaronCGoidel) [inline components/ScrollingAvatars/ScrollingAvatarsContainer.tsx:37]: I get what's going on here, but I'd like some documentation in this function body in general. It's a bit hard to follow, especially compared to the rest of the code in this PR

File: scss/index.scss [hunk 6]
```diff
@@ -1,5 +1,11 @@
 // our project themes
-@import "./theme/colors";
+@import './theme/colors';
 
 // import your custom scss files below here
-@import "./heroNotificationBox.scss";
+@import './heroNotificationBox.scss';
```
Comment (AaronCGoidel) [inline scss/index.scss:5]: In general, try not to format code not touched by the PR. I assume this was an auto-formatter

File: components/ScrollingAvatars/ScrollingAvatarsContainer.tsx (hunk 3 above)
Comment (TheFatPanda97) [reply components/ScrollingAvatars/ScrollingAvatarsContainer.tsx:10]: hmm I'll just remove it then

File: scss/index.scss (hunk 6 above)
Comment (TheFatPanda97) [reply scss/index.scss:5]: I added an scss file that's why I formatted it 😅

File: pages/index.tsx [hunk 7]
```diff
@@ -18,6 +19,41 @@ const Home: FC = () => (
...
+          position: 'President',
+          imgSrc: 'https://i.pravatar.cc/150?img=5',
+        },
+        {
+          name: 'Shang Chi',
+          position: 'President',
+          imgSrc: 'https://i.pravatar.cc/150?img=6',
+        },
+      ]}
+    />
```
Comment (NerajM) [inline pages/index.tsx:56]: Should we have this list as a prop lie this, or is there a better way to store it? i.e. a JSON file?

File: components/ScrollingAvatars/Avatar.tsx [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+interface IProps {
+  /** width and height of avatar */
+  size?: number;
+  className?: string;
+  imgSrc: string;
+  onMouseEnter?: () => void;
+  onMouseLeave?: () => void;
+}
+
+// eslint-disable-next-line object-curly-newline
```
Comment (nigow) [inline components/ScrollingAvatars/Avatar.tsx:13]: you can remove it
